system-test/system-test-script.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+#Stopping the script if any command fails
+set -euo pipefail
+export myvar="system-test/logfile"
+cd ..
+ echo "Starting collection roller....."
+bin/collection-roller 2>&1 > $myvar &
+#Getting the process id of collection roller
+pid1=$!
```

Comment:
how about `collection_roller_pid`, it would be more descriptive.

Reply:
fixed.